planning/behavior_velocity_planner/src/node.cpp

```diff
@@ -302,6 +302,9 @@ void BehaviorVelocityPlannerNode::onVehicleVelocity(
       break;
     }
 
+    if (!planner_data_.velocity_buffer.empty()) {
```

Comment:
```suggestion
    if (planner_data_.velocity_buffer.empty()) {
```

The condition is wrong?